### PR TITLE
add updates link to Finance and Meeting notes pages

### PR DESCRIPTION
**Pull request by KellyAH**
- For https://github.com/zinc-collective/www.zinc.coop/issues/103

Add https://opencollective.com/zinc-community/updates link to Finance and Meeting pages.

finance page
meeting notes

### CODE REVIEW

File: src/financial-overview/index.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Financial Overview for Zinc Collective, LLC
 ---
 We publish our Profit and Loss and Balance Sheet reports on a quarterly and annual basis as part of our commitment to public and community governance and oversight.
 
+See the latest updates [here](https://opencollective.com/zinc-community/updates).
+
```
Comment (KellyAH) [inline src/financial-overview/index.md:9]: It would be nice if I could get this link to open in a new tab. But right now it doesn't.

Comment (anaulin) [reply src/financial-overview/index.md:9]: I think you might be able to pass in the option to open in a new window/tab with:
```
[link](url){:target="_blank"}
```

**Review by anaulin — APPROVED** [review]

File: src/financial-overview/index.md (hunk 1 above)
Comment (KellyAH) [reply src/financial-overview/index.md:9]: darn. it didn't work. had to go with straight HTML ce2b61de5c0bdd7ddf415a698ff72af05b7f109b

**Comment by KellyAH** [comment]
🚀 deployed and ✅ verified on prod 💃

**Comment by zspencer** [comment]
Thank you!!!!

**Pull request merged**